sound/soc/sof/control.c

```diff
@@ -52,9 +52,10 @@ int snd_sof_volume_get(struct snd_kcontrol *kcontrol,
 	}
 
 	/* get all the mixer data from DSP */
-	snd_sof_ipc_get_comp_data(sdev->ipc, scontrol, SOF_IPC_COMP_GET_VALUE,
-				  SOF_CTRL_TYPE_VALUE_CHAN_GET,
-				  SOF_CTRL_CMD_VOLUME);
+	snd_sof_ipc_set_get_comp_data(sdev->ipc, scontrol,
+				      SOF_IPC_COMP_GET_VALUE,
+				      SOF_CTRL_TYPE_VALUE_CHAN_GET,
+				      SOF_CTRL_CMD_VOLUME);
```

Comment:
can we have all the changes introducing snd_sof_ipc_set_get_comp_data in a separate patch - this is orthogonal to large messages.

Reply:
@plbossart ok, I will refactor the patch set. This was coming from @ranj063 connents that we could unify the set&get calls because 95 percent of the code is the same.